build/Dockerfile.ci

```diff
@@ -68,6 +54,24 @@ COPY containernetworking-plugins/ .
 ENV CGO_ENABLED=0
 RUN ./build_windows.sh
 
+# Build WMCO
+WORKDIR /build/windows-machine-config-operator
+# Copy files and directories needed to build the WMCO binary
+COPY build build
```

Comment:
is this the only folder that is required to be transferred at the end to solve the dirty tag issue, or do we need something else too?

Reply:
We need to transfer .gitignore and move building the WMCO operator after building all submodules. That will remove the dirty tag.